data/card_catalogue/upgradeable-1.json

```diff
@@ -204,7 +264,10 @@
     "upgrade": "Pray",
     "tier": "blue",
     "details": {
-      "rules": "Change the value of thrown d20 or the next thrown d20 by 2",
+      "rules": {
+        "active": "Change the result of a roll by 2.",
```

Comment:
following the change to boost c8278c49a16c20ecea106ea6ce86b46bdb36d3c4 and fluster c8278c49a16c20ecea106ea6ce86b46bdb36d3c4